contracts/MarketRegistry.sol

```diff
@@ -124,6 +130,11 @@ contract MarketRegistry is IMarketRegistry, ClearingHouseCallee, MarketRegistryS
         emit MarketMaxPriceSpreadRatioChanged(baseToken, ratio);
     }
 
+    function setFeeDiscountRatio(address trader, uint24 discountRatio) external checkRatio(discountRatio) onlyOwner {
```

Comment:
can add a check that `discountRatio < _ONE_HUNDRED_PERCENT_RATIO`

Reply:
We already have `checkRatio` modifier, but it didn't use `_ONE_HUNDRED_PERCENT_RATIO`, although these two value is the same